PyTorchCustomOperator/README.md

```diff
@@ -0,0 +1,236 @@
+## Implementing Custom Operators in PyTorch, ONNX, and ONNX Runtime
+
+This document describes the required steps for extending TorchScript with a custom operator, exporting the operator to ONNX format, and adding the operator to ONNX Runtime for model inference.
+Although the ```torch``` module provides a broad set of tensor operators, TorchScript enables users to design and implement their custom (C++ or CUDA) function and register it as a new operator.
+To export such a custom operator to ONNX format, the custom op registration ONNX API enables users to export a custom TorchScript operator using a combination of existing and/or new custom ONNX ops.
+Once the operator is converted to ONNX format, users can implement and register it with ONNX Runtime for model inference. This document explains the details of this process end-to-end, along with an example.
+
+### Required Steps
+
+  - [1](#step1) - Adding the custom operator implementation in C++ and registering it with TorchScript
+  - [2](#step2) - Exporting the custom Operator to ONNX, using:
+  <br />             - a combination of existing ONNX ops
+  <br />              or
+  <br />              - a custom ONNX Operator
+  - [3](#step3) - Adding the custom operator implementation and registering it in ONNX Runtime

```

Comment:
Step 3 is only required when Step2 uses a custom ONNX op. Can we explicitly make a statement here?